### PR TITLE
Update project descriptions for Spring 2019

### PR DESCRIPTION
**Pull request by xinhaoz**
- added Travelwise project
- updated Elevate description
- moved PFC to past projects

### CODE REVIEW

**Comment by silastsui** [comment]
I think Susan updated the description for Datassist now. After those changes, lgtm!

File: src/app/projects-page/projects-page.component.html [hunk 1]
```diff
@@ -25,10 +25,14 @@ <h1>current</h1>
           <h2 class="uppercase">Elevate</h2>
         </div>
         <div class="u-margin-bottom-10">
-            Elevate is a Waterloo-based social enterprise that is tackling the lack of inclusion for individuals with cognitive disabilities. Without enough accessible resources available, Elevate is developing a game-based learning platform that teaches independent living skills in an engaging and low-cost way.
+            Elevate is a game based e-platform that empowers individuals with Down syndrome to develop independent living skills. 
+            They were selected as the winner by a team of esteemed judges for the $5K Vlocity accessibility prize at the Velocity
```
Comment (sherryhli) [inline src/app/projects-page/projects-page.component.html]: Velocity typo?

File: src/app/projects-page/projects-page.component.html [hunk 2]
```diff
@@ -25,10 +25,14 @@ <h1>current</h1>
...
+            Elevate is a game based e-platform that empowers individuals with Down syndrome to develop independent living skills. 
+            They were selected as the winner by a team of esteemed judges for the $5K Vlocity accessibility prize at the Velocity 
+            Fund Finals. Elevate’s game-based e-learning platform will include modules like going to the bank and taking the bus.
         </div>
         <div>
-            For Blueprint, we’re focusing on helping users and their supervisors track achievements and engagement for better understanding of how the game platform is impacting their progress. We’re conducting research, crafting the online experience, and building the backend and interface for this dashboard feature.
+            Blueprint is helping Elevate build a platform that will track each individual's performance in the game and provide 
+            caregivers with insights on the individual's progress. This will serve as a comprehensive tool to administer the 
+            caregivers’ players as well as provide them with an informative dashboard.
```
Comment (sherryhli) [inline src/app/projects-page/projects-page.component.html]: I know we weren't responsible for the content but any chance we could do a small reword here? Find this sentence a bit difficult to follow.

File: src/app/projects-page/projects-page.component.html [hunk 3]
```diff
@@ -37,10 +41,14 @@ <h2 class="uppercase">Elevate</h2>
           <h2 class="uppercase">Datassist</h2>
         </div>
         <div class="u-margin-bottom-10">
-            <a href="https://idatassist.com/">Datassist</a> is a company that provides data science services to non-profits  through data visualization to help them tell stories with their results. They’re building a new project called We All Count, an equity tool for data science that aims to identify biases in datasets and algorithms.
+            Located in Toronto, <a href="https://idatassist.com/">Datassist</a> is an organization of staticians and data
```
Comment (sherryhli) [inline src/app/projects-page/projects-page.component.html]: Statistician typo?

File: src/app/projects-page/projects-page.component.html (hunk 2 above)
Comment (xinhaoz) [reply src/app/projects-page/projects-page.component.html]: Good catch on the typos 👏 Elevate's PL has changed the description to be more clear 👍

**Pull request merged**